front-page.php

```diff
@@ -0,0 +1,111 @@
+<?php
+get_header();
+?>
+
+<?php
+  $site_options = get_site_option('_igv_site_options');
+
+  if (!empty($site_options['_igv_frontpage_postit_image_id'])) {
+?>
+<div id="postit">
+  <?php echo wp_get_attachment_image($site_options['_igv_frontpage_postit_image_id'], 'thumbnail', false, 'class=postit-image'); ?>
+  <div id="postit-dot" class="dot"></div>
+</div>
+<?php
+  }
+?>
+
+<main id="main-content">
+  <section id="posts">
+    <div class="container">
+
+<?php
+$now = time();
+
+$current_args = array(
+	'post_type' => array( 'exhibition' ),
```

Comment:
mixed tabs and spacing

Reply:
ah its cause copy pasting from browser. 

just fix it...